core/src/i18n/en.yaml

```diff
@@ -1005,9 +1005,19 @@ extensibility:
     old-version: There is a newer version of the extensibility configuration. Please migrate to the newest version.
     unnown-version: Your extensibility configuration version is no longer supported, but cannot be migrated. For more information, see <0>Extensibility documentation</0>.
     unsupported-version: Your extensibility configuration version is no longer supported. Please migrate to the newest version.
+  sections:
+    details: Details View
+    form: Form Fields
+    lang-translations: Translations ({{lang}})
+    list: List Columns
+    relations: Resource Relations
+    resource: Resource Basics
+    translations: Translations
```

Comment:
do we need both `translations` and `lang-translations`?

Reply:
yes, because one is generic, and the other has the extra language